### PR TITLE
helium wallet migration guide

### PR DESCRIPTION
**Pull request by samgutentag**

### CODE REVIEW

File: docs/wallets/wallets.mdx [hunk 1]
```diff
@@ -9,68 +10,199 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
 
 <img className="docsheader" src={useBaseUrl('img/wallets/header.png')} />
 
-# Helium Wallets
+## Helium Is Moving To Solana
 
-In blockchain projects, various wallet applications exist to help users manage their own private
-keys, enabling them to sign transactions and transfer assets.
+All Hotspots on the Helium Network with an animal name will automatically become a
```
Comment (cokes518) [inline docs/wallets/wallets.mdx]: this feels like a very abrupt introduction to "Helium is moving to Solana". Can there be a little more preamble?

File: docs/wallets/wallets.mdx [hunk 2]
```diff
@@ -9,68 +10,199 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
+Makers must push this mandatory update to their fleet of Hotspots. Failure to do so will mean their
+customers cease to mine and stop earning rewards once the migration is complete.
 
-- Multiple Account Support
-- Ledger Balance
-- Address Book
-- Verify your Helium Identity
-- Helium Vote
-- Multiple Recipient Payment Support
+### Hotspots Will Being To Earn IOT
```
Comment (cokes518) [inline docs/wallets/wallets.mdx]:
```suggestion
### Hotspots Will Earn IOT
```

File: docs/wallets/wallets.mdx [hunk 3]
```diff
@@ -9,68 +10,199 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
+The existing [Helium Hotspot app](/wallets/app-wallet) will be restricted to only management of
+Original Helium Hotspots in two phases.
+
+**Phase 1:**
+
+> ETA December 19, 2022
+
+- Hotspot App users will be migrated to use the [Helium Wallet app](/wallets/helium-wallet-app) for
+  Helium Wallet management.
+- New Hotspots will no longer be able to be onboarded using the Helium App, and will instead to app
```
Comment (cokes518) [inline docs/wallets/wallets.mdx]:
```suggestion
- Users can no longer onboard new Hotspots using the Helium App. All new Hotspots will be directed to use the Hotspot's Maker app, web dashboard, or contact email support (in the event that a Maker app is not available). 

:::note
All Makers had a deadline to launch their Maker app by March of 2022. Makers without an app or web dashboard either have an app in development or have not responded to requests for additional updates. 
:::


```

File: docs/wallets/wallets.mdx [hunk 4]
```diff
@@ -9,68 +10,199 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
+- Hotspot management will be handled by Maker apps.
+- Original Helium Hotspots will be managed by the Helium App.
+- All remaining wallet functionality will use the Helium Wallet app.
+
+:::note What If I Miss The Migration Deadline?
+
+Users that do not transition to the Helium Wallet app prior to the Solana migration will still be
+able to access their seed phrase and transition to the Helium Wallet app but will not be able to
+submit transactions in the Helium app.
+
```
Comment (cokes518) [inline docs/wallets/wallets.mdx:138]:
```suggestion
Seed phrase export functionality will remain in the app for at least 1 year post-migration.
```

File: docs/wallets/wallets.mdx [hunk 5]
```diff
@@ -9,68 +10,199 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
+The [Helium app](/wallets/app-wallet) will require a migration to the
+[Helium Wallet app](/wallets/helium-wallet-app). Migration is necessary to interact with your HNT,
+DC, MOBILE, IOT, and Hotspots.
+
+To start, ensure your [Helium app](/wallets/app-wallet) is updated and that you have downloaded the
+latest Helium Wallet app.
+
+1. Open the [Helium app](/wallets/app-wallet) (minimum v4.3.0)
+2. Tap `Export 12 Word Seed Phrase` in the migration popup
```
Comment (cokes518) [inline docs/wallets/wallets.mdx]:
```suggestion
2. Tap `Start Migration` in the migration popup
```

File: docs/wallets/wallets.mdx [hunk 6]
```diff
@@ -9,68 +10,199 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
+3. The [Helium Wallet app](/wallets/helium-wallet-app) will automatically open. Tap `Import`
```
Comment (cokes518) [inline docs/wallets/wallets.mdx]:
```suggestion
3. Confirm the Wallet account you want to import, then tap `Export to Wallet App`. The [Helium Wallet app](/wallets/helium-wallet-app) will automatically open and complete the migration.
```

File: docs/wallets/wallets.mdx [hunk 7]
```diff
@@ -9,68 +10,199 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
+4. Now your HNT, DC, MOBILE, and Hotspots have successfully moved to the Solana Blockchain.
```
Comment (cokes518) [inline docs/wallets/wallets.mdx]:
```suggestion
4. Now your HNT, DC, MOBILE, and Hotspots have successfully moved to the Helium Wallet App and ready for the Solana migration, which will happen automatically in January 2023.
```

File: docs/wallets/wallets.mdx [hunk 8]
```diff
@@ -9,68 +10,199 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
+
+You will use the [Helium Wallet app](/wallets/helium-wallet-app) moving forward. You will locate
+Hotspot NFTs in the NFT Collectibles section, and token balances are visible on the account page.
```
Comment (cokes518) [inline docs/wallets/wallets.mdx]:
```suggestion
Hotspot NFTs in the NFT Collectibles section once Helium on Solana goes live. Token balances are visible on the account page.
```

File: docs/wallets/wallets.mdx [hunk 9]
```diff
@@ -9,68 +10,199 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
+
+To start, ensure your [Helium app](/wallets/app-wallet) is updated and that you have downloaded the
+latest Helium Wallet app.
+
+1. Open the [Helium app](/wallets/app-wallet) (minimum v4.3.0)
+2. Tap `Export 12 Word Seed Phrase` in the migration popup
+3. The [Helium Wallet app](/wallets/helium-wallet-app) will automatically open. Tap `Import`
+4. Now your HNT, DC, MOBILE, and Hotspots have successfully moved to the Solana Blockchain.
+
+You will use the [Helium Wallet app](/wallets/helium-wallet-app) moving forward. You will locate
```
Comment (cokes518) [inline docs/wallets/wallets.mdx]:
```suggestion
You will use the [Helium Wallet app](/wallets/helium-wallet-app) moving forward. You can find
```

File: docs/wallets/wallets.mdx [hunk 10]
```diff
@@ -9,68 +10,199 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
+Hotspot NFTs in the NFT Collectibles section, and token balances are visible on the account page.
+You can also create multiple accounts in the [Helium Wallet app](/wallets/helium-wallet-app).
+
+Your public wallet address will change, but your private key and 12-word seed phrase will remain the
```
Comment (cokes518) [inline docs/wallets/wallets.mdx]:
```suggestion
Your public wallet address will change post-migration, but your private key and 12-word seed phrase will remain the
```

File: docs/wallets/wallets.mdx (hunk 1 above)
Comment (samgutentag) [reply docs/wallets/wallets.mdx]: preamble added, may need some more work. but also didn't want to deviate too far off topic from Wallet stuff to general L1 migration stuff

File: docs/wallets/helium-wallet-app.mdx [hunk 11]
```diff
@@ -10,9 +10,43 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
+          src={useBaseUrl('img/app-store-badges/google-play-store-badge-en.svg')}
+        />
+      </a>
+    </div>
+  </div>
+</div>
+
+:::
+
 The Helium Wallet app is the new trusted way to manage your tokens and identity on the Helium
```
Comment (cokes518) [inline docs/wallets/helium-wallet-app.mdx]:
```suggestion
The Helium Wallet app is the trusted way to manage your tokens and identity on the Helium
```

File: docs/wallets/helium-wallet-app.mdx [hunk 12]
```diff
@@ -10,9 +10,43 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
 Blockchain. This new Wallet App is not meant to manage Hotspots or Validators.
```
Comment (cokes518) [inline docs/wallets/helium-wallet-app.mdx]:
```suggestion
Blockchain. 
```

File: docs/wallets/wallets.mdx [hunk 13]
```diff
@@ -9,68 +10,245 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
+
+The [Helium app](/wallets/app-wallet) will require a migration to the
+[Helium Wallet app](/wallets/helium-wallet-app). Wallet migration is necessary to interact with your
+HNT, DC, MOBILE, IOT, and Hotspots on the Solana blockchain.
+
+To start, ensure your [Helium app](/wallets/app-wallet) is updated and that you have downloaded the
+latest Helium Wallet app.
+
+1. Open the [Helium app](/wallets/app-wallet) (minimum v4.3.0)
+2. Tap `Start Migration` in the Wallet migration popup
```
Comment (cokes518) [inline docs/wallets/wallets.mdx]: screenshots sent offline.

File: docs/wallets/wallets.mdx [hunk 14]
```diff
@@ -9,68 +10,245 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
+remain the same.
+
+### For Ledger Wallet Users
+
+Ledger users will need to update their Helium Ledger App to a version that's pending release. This
+upcoming version will use the Solana private key derivation path that moves all your tokens and
+balances over to Solana. You may need to uninstall any Solana Ledger Apps during the Wallet
+migration process.
+
+Your public key will change on Solana and the private key will also need to be updated.
```
Comment (cokes518) [inline docs/wallets/wallets.mdx]:
```suggestion
Your public key will change on Solana but the private key will remain the same.
```

File: docs/wallets/wallets.mdx [hunk 15]
```diff
@@ -9,67 +10,297 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
...
+
+Check out the [Solana Documentation](https://docs.solana.com/wallet-guide/cli) for more information
+on Solana Command Line Wallets.
+
+:::
+
+The CLI Wallet will have its feature set pared down as there are many other Solana-compatible CLI
+Wallets. The existing Helium CLI Wallet will be reduced to the following two functions:
+
+1. Export Seed Phrase to import into a Solana CLI Wallet (or the Helium Wallet app)
```
Comment (cokes518) [inline docs/wallets/wallets.mdx]:
```suggestion
1. Export Seed Phrase or private key to import into a Solana CLI Wallet (or the Helium Wallet app)
```

**Review by cokes518 — APPROVED** [review]

**Pull request merged**